tools/build/build.js

```diff
@@ -109,17 +154,67 @@ const taskDm = new Task('dm')
         || 'DreamMaker'
       );
     })();
-    await exec(dmPath, ['tgstation.dme']);
+    if (injectedDefines.length) {
+        const injectedContent = injectedDefines
+          .map(x => `#define ${x}\n`)
+          .join('')
+        // Create mdme file
+        fs.writeFileSync(`${DME_NAME}.mdme`, injectedContent)
```

Comment:
Why are you using the blocking methods if this function is going to be async anyway?

Reply:
Why are we going in a blocking sequence at all when some of these (like renaming dmb and rsc) should be able to be performed in parallel? You could combine everything that can run in parallel with each other with `Promise.all`.

Reply:
There's no need to work with these files in parallel, also async fs creates overhead.